src/catalog/src/table_source.rs

```diff
@@ -51,6 +55,7 @@ impl DfTableSourceProvider {
         disallow_cross_catalog_query: bool,
         query_ctx: &QueryContext,
         plan_decoder: SubstraitPlanDecoderRef,
+        enable_ident_normalization: bool,
```

Comment:
Does it make sense to handle this thing here? I'm wondering if it's possible to hide this "detail" in parser

Reply:
I am afraid not because the old columns are retrieved from the logical plan's schema, and it's already in `operator`.

Second, this option uses sql parser option value in `SessionState`.